packages/theming/theme-tokens/src/getMacOSAliasTokens.ts

```diff
@@ -0,0 +1,15 @@
+import * as macOSLightAliasTokens from './light-macos/tokens-alias';
+import * as macOSDarkAliasTokens from './dark-macos/tokens-alias';
+import * as hcAliasTokens from './highContrast/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+
+export function getMacOSAliasTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return macOSLightAliasTokens.default;
+  } else if (mode === 'dark') {
+    return macOSDarkAliasTokens.default;
+  } else if (mode === 'highContrast') {
+    return hcAliasTokens.default;
+  }
```

Comment:
nit: we can `assertNever` here to make sure we get a build failure is a new AppearanceOption is added

Reply:
Added, I also removed the default case as it is unreachable I think...

Reply:
`assertNever` was added to check that we never reach theoretically unreachable cases, so I think we would want to add that.

Reply:
jk, it's there. My bad